src/schema/v1/artwork/index.ts

```diff
@@ -601,31 +615,36 @@ export const ArtworkType = new GraphQLObjectType<any, ResolverContext>({
             artwork.domestic_shipping_fee_cents == null &&
             artwork.international_shipping_fee_cents == null
           )
-            return "Shipping, tax, and service quoted by seller"
+            return "Shipping, tax, and additional fees quoted by seller"
           if (
             artwork.domestic_shipping_fee_cents === 0 &&
             artwork.international_shipping_fee_cents == null
           )
-            return "Free shipping within continental US only"
+            return "Free domestic shipping"
           if (
             artwork.domestic_shipping_fee_cents === 0 &&
             artwork.international_shipping_fee_cents === 0
           )
             return "Free shipping worldwide"
-          var domesticShipping = amount(
-            ({ domestic_shipping_fee_cents }) =>
-              domestic_shipping_fee_cents || null
-          ).resolve(artwork, { precision: 0 })
```

Comment:
I believe that instead of refactoring the `money` helper here to return a display helper, you can pass `{ precision: 0, symbol: symbolFromCurrencyCode(artwork.price_currency)  }` and it should resolve how you want it to.